src/main/java/com/jalasoft/webservice/controller/VideoController.java

```diff
@@ -1,9 +1,81 @@
+/**
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ * ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
```

Comment:
Insert space between copyright and the package

Reply:
Done.